chainer/functions/activation/sigmoid.py

```diff
@@ -76,7 +76,6 @@ def sigmoid(x, use_cudnn=True):
 
     .. admonition:: Example
 
-        It maps the input values into the range of :math:`[0, 1]`.
```

Comment:
@keisuke-umezawa Sorry, could you check this again? Somehow it seems to be reverted before you added lines here.